students/YingGuo/session02/series.py

```diff
@@ -0,0 +1,80 @@
+
+
+def fibonacci(n):
+    """The Fibonacci Series is a numeric series starting with the integers 0 and 1.
+    In this series, the next integer is determined by summing the previous two. 
+    This function is to write Fibonacci series. N is the length of series. N is an integer starting from 0"""
+   
+    list1 = list()
+
+    for i in range(n+1):
+        if i == 0:
+            list1.append(0)
+        elif i == 1:
+            list1.append(1)
+        else:
+            x = list1[i-2]+list1[i-1]
+            list1.append(x)
```

Comment:
This works, but it'd be better if you don't need to use a list of many numbers just to compute the final value. No need to fix, and the solutions that'll be posted are not most efficient anyway, so this is fine.